drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -379,7 +378,7 @@ static void gps_thread(int dev_ptr)
 
 				notify_event(dev, &evt);
 
-				k_delayed_work_cancel(&drv_data->blocked_work);
+				k_work_cancel_delayable(&drv_data->blocked_work);
```

Comment:
what happens if this fails?

Reply:
Switched to `k_work_cancel_delayable_sync`.